### DISCUSSION TITLE
Filter based on multiple condition with `where` query parameter [Feb 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5210] [Discord Threads]
> 🙋‍♂️ **Joecode**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### In UI with the search field at the moment you can only use one column based filter but with the api you can do something like you shared `/api/v1/data/<organization>/<project>/<table>/views/<view>?offset=0&limit=25&where=(<Column>,like, <search term>)` and the column names need to be hardcoded  
#  
> 👨‍💻 **pranavxc**  
#### If you want multiple filter you can use the Filter menu  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1065365031325290667)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
